source/hide-files-on-github.tsx

```diff
@@ -1,4 +1,3 @@
-import 'webext-dynamic-content-scripts';
```

Comment:
The build was completely broken due to this, it was supposed to be removed when updated to v6